### PR TITLE
fix: replace hash icon to arrow down for filter select

### LINKED ISSUE #1812: Add visual indication that topics can be changed
I've received feedback a handful of times now that users didn't know they can change the topic.

Currently: 
image

Proposed change:
image

Maybe adding a down triangle selector is a good indication that it can be changed.

### PR DESCRIPTION
**Pull request by Gourav2609**
## Description

i have done the changes suggested by @BekahHW  in PR #1816  
and issue #1812 

Please review this PR

# Before

273069319-c8786858-b1c2-4430-ae1b-172eb5974d8c

# After

image

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [1 ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/115522947/59e9f461-5600-420b-bb92-eef7a556d3ca

## Added tests?

- [ ] 👍 yes
- [ 1] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Gourav2609** [comment]
@BekahHW  any suggestions?

File: components/molecules/FilterCardSelect/filter-card-select.tsx [hunk 1]
```diff
@@ -4,10 +4,11 @@ import Image from "next/image";
 import { BsFillCheckCircleFill } from "react-icons/bs";
 
 import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "components/atoms/Select/select";
-import hashIcon from "../../../img/icons/hash.svg";
+// import hashIcon from "../../../img/icons/hash.svg";
```
Comment (bdougie) [inline components/molecules/FilterCardSelect/filter-card-select.tsx:7]: dead code needs to be removed.

**Review by bdougie — DISMISSED** [review]
Where are you sourcing the SVG? The # looks rather big.

File: components/molecules/FilterCardSelect/filter-card-select.tsx (hunk 1 above)
Comment (Gourav2609) [reply components/molecules/FilterCardSelect/filter-card-select.tsx]: @bdougie Please checkout the latest commit . 
i have remove the comment code and fixed the hash.svg ....
Earlier i was using a different svg code from ....but now i have fixed it with the same code as it was before

**Comment by bdougie** [comment]
Mobile screen doesn't have the arrow pointing down.  image

**Review by jpmcb — COMMENTED** [review]
This shows up as a left facing arrow on Safari mobile: is that intended?

image

**Comment by Gourav2609** [comment]
> Mobile screen doesn't have the arrow pointing down. image

I tried to replicate the error but could not do so
image

**Comment by jpmcb** [comment]
Screenshot 2023-10-11 at 8 41 02 AM

It must be something with how Safari renders these icons. Here is Safari desktop with the same problem.

**Comment by nickytonline** [comment]
I believe we want to be using this icon, https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg @Gourav2609

**Comment by Gourav2609** [comment]
> I believe we want to be using this icon, https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg @Gourav2609

sure ....i will revert the changes and look for a solution

**Comment by Gourav2609** [comment]
@bdougie @jpmcb i have replaced svg with png. this should fix the safari error

File: img/icons/chevron-left.svg
Comment (nickytonline) [inline img/icons/chevron-left.svg:1]: Refactor (blocking): These icons should not be part of the pull request. The icon required for the fix is already in the project.

https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg

**Review by nickytonline — CHANGES_REQUESTED** [review]

File: img/icons/chevron-left.svg
Comment (Gourav2609) [reply img/icons/chevron-left.svg:1]: I have removed inverted-triangle.png and currently using chevron-down.svg that was already there in the img .

**Comment by OgDev-01** [comment]
Looks good on my end(chrome)... i'll leave it up to @nickytonline and @jpmcb to drop their review based on browser support 👍
image

**Comment by Gourav2609** [comment]
@OgDev-01 Thanks for the review

i would also request @nickytonline @bdougie do suggest any changes

Thank in advance

**Review by nickytonline — CHANGES_REQUESTED** [review]
Refactor (blocking): There's changes to the chevron-left icon which shouldn't be a part of this PR.

I'll take it for a spin, but once the above icon changes are removed, this should be good.

**Comment by Gourav2609** [comment]
> Refactor (blocking): There's changes to the chevron-left icon which shouldn't be a part of this PR.
> 
> I'll take it for a spin, but once the above icon changes are removed, this should be good.

i changed the code for already present `chevron-left.svg` in my past commits trying different ways....in my most recent commit `fixed icons` ... i have added no new icons or svgs...used the already present `chevron-down.svg `

Do i still need to do any changes...Please let me know @bdougie @nickytonline @OgDev-01

**Comment by nickytonline** [comment]
@Gourav2609, the chevron-left icon changes are still in the pull request. What I suggested won't work because your pull request is using the beta branch in your fork.

You'll need to revert it instead to an a version prior to your changes, e.g.

1. `git checkout 4d240a84 -- img/icons/chevron-left.svg`
1. `git commit -m "chore: reverted changes to chevron-left icon"`
1. `git push origin your_branch_name`

**Comment by Gourav2609** [comment]
@nickytonline Thanks so much for the help

Can u please review it

**Review by nickytonline — DISMISSED** [review]
Thanks @Gourav2609. Looks good on Chromium, Firefox & Safari

**Chromium**

CleanShot 2023-10-16 at 16 47 13

**Firefox**

CleanShot 2023-10-16 at 16 48 28

**Safari**

CleanShot 2023-10-16 at 16 49 14

**Comment by Gourav2609** [comment]
Thanks @nickytonline 

any suggestions from @bdougie @jpmcb

**Comment by nickytonline** [comment]
> Thanks @Gourav2609. Looks good on Chromium, Firefox & Safari

Looking at the icon placement actually, it could be closer to the right. That might be unrelated to your change though. Just taking a peek.

**Comment by Gourav2609** [comment]
In issue #1812 , @bdougie pointed about the icon being slightly towards right and wanted it to be aligned properly...in response to that i aligned it to center...
No prob if you want i can shift it towards right🙂

**Review by nickytonline — APPROVED** [review]
Alright, let's go ahead with this. Maybe going forward we use your new dropdown component @OgDev-01?

**Comment by OgDev-01** [comment]
> Maybe going forward we use your new dropdown component @OgDev-01?

Ofcourse 👍. we can merge this as is

**Review by OgDev-01 — APPROVED** [review]
🍕

**Comment by Gourav2609** [comment]
thanks @bdougie @nickytonline @OgDev-01  and everyone...looking forward to contribute more😊

**Pull request merged**